.docs/content/getting-started/installation/kubernetes.md

```diff
@@ -1,58 +1,101 @@
-# Kubernetes deployment
+# Kubernetes Installation
 
-If you want to deploy ArmoniK on Kubernetes, this section is for you.
+If you already have a K3s installation, start by uninstalling it properly.
 
+## Step 1: Stop the K3s service
 
-## Clone ArmoniK
+```bash
+sudo systemctl stop k3s
+```
 
+## Step 2: Run the uninstall script
 
-First, clone the ArmoniK repository (inside your home directory from WSL2 Ubuntu distribution):
+K3s provides an uninstall script to remove all installed components.
 
-```bash [shell]
-git clone https://github.com/aneoconsulting/ArmoniK.git
+```bash
+sudo /usr/local/bin/k3s-uninstall.sh
 ```
 
-## Deploy
 
-To launch the deployment, go to the [`infrastructure/quick-deploy/localhost`](https://github.com/aneoconsulting/ArmoniK/tree/main/infrastructure/quick-deploy/localhost) directory:
 
-If you want to deploy on AWS, go to the dedicated section on [`AWS`](https://github.com/aneoconsulting/ArmoniK/tree/main/infrastructure/quick-deploy/aws)
+## Step 3: Clean up remaining files (optional)
 
-Execute the following command:
+After the uninstall script, you can also remove any remaining K3s and Kubernetes files:
 
 ```bash
-make
+sudo rm -rf /etc/rancher/k3s
+sudo rm -rf /var/lib/rancher
+sudo rm -rf /var/lib/kubelet
+sudo rm -rf ~/.kube
 ```
 
-or
+>**Note**: This step completely removes all K3s configuration and data. Only use this if you want a completely clean installation.
+
+# K3s Installation
+
+## Step 1: Run the installation script
 
+### Option A: Use ArmoniK's installation script (Recommended)
+Use the installation script available here: [infrastructure/utils/scripts/installation/prerequisites/install-k3s.sh](https://github.com/aneoconsulting/ArmoniK/blob/main/infrastructure/utils/scripts/installation/prerequisites/install-k3s.sh)
+
+### Option B: Use the official installation script
+You can also use the official installation script to install K3s:
 ```bash
-make deploy
+curl -sfL https://get.k3s.io | sh -s - --write-kubeconfig-mode 644 --docker
 ```
 
-## Configuration
 
-All parameters are contained in [`parameters.tfvars`](https://github.com/aneoconsulting/ArmoniK/blob/main/infrastructure/quick-deploy/localhost/parameters.tfvars)
 
+## Step 2: Configure kubeconfig for your user (optional)
```

Comment:
Add a line stating that this should not be necessary if the user employs the recommended installation script